strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/providers/layout/NugetHierarchicalLayoutProvider.java

```diff
@@ -173,6 +173,14 @@ protected ArtifactOutputStream decorateStream(RepositoryPath path,
         }
         
     }
+    
+    @Override
+    protected void addArtifactToIndex(RepositoryPath path)
```

Comment:
This method should not exist in any of these implementations. We don't need empty methods that confuse things unnecessarily.

Also, the indexing is just for the Maven repositories. It's not for Nuget, for example. The `addToIndex()` is just for the Maven layout provider.